src/google/protobuf/compiler/parser.cc

```diff
@@ -86,7 +86,7 @@ TypeNameMap MakeTypeNameTable() {
 const TypeNameMap kTypeNames = MakeTypeNameTable();
 
 // Camel-case the field name and append "Entry" for generated map entry name.
-// e.g. map<KeyType, ValueType> foo_map => FooMapEntry
+// e.g. std::map<KeyType, ValueType> foo_map => FooMapEntry
```

Comment:
Here and below in this file, std:: shouldn't be prepended since these are comments in proto syntax.